### ISSUE TITLE
The Difference Between 3x3 Autotiling and 3x3 Minimal Autotiling

### ISSUE THREAD

**Issue by Capital-EX** [godotengine/godot-docs#3070]
I would like to request document explaining the difference between 3x3 Autotiling and 3x3 Minimal Autotiling.  Currently, the docs have no explanation on what they do.  However, they have very different behaviors.

https://docs.godotengine.org/en/3.1/classes/class_tileset.html#enumerations

**Comment by cbscribe** [comment]
Agreed this would be helpful - there's always a lot of confusion between the two. One thing we'd need to make this would be a couple of good example tile sets, which are CC licensed. Does anyone have some suggestions?

**Comment by Calinou** [comment]
> One thing we'd need to make this would be a couple of good example tile sets, which are CC licensed.

[This one](https://kenney.nl/assets/tower-defense-top-down) or [this one](https://kenney.nl/assets/platformer-art-extended-tileset)?

**Comment by cbscribe** [comment]
More specifically a 3x3 one. 3x3 minimal tilesets (48 tiles) are relatively easy to find, but 3x3 requires over 200 tiles, so they're a lot more time consuming to draw.

**Comment by dalexeev** [comment]
> but 3x3 requires over 200 tiles

If you use wildcard bit, you can greatly reduce this number.

**Comment by golddotasksquestions** [comment]
Wildcards need documentation:
https://github.com/godotengine/godot-docs/issues/3491
Add complete Tileset reference for download to the documentation:
https://github.com/godotengine/godot-docs/issues/3316
